Da-Bois-Tracking/Tracking-App
# Added Python + Flask

### In this PR, I'm adding/changing the following:

- Starting development of the backend with Python
- Installation of Flask

### Please do the following to test my code:

1. Clone the repo, if you haven't already
   `git clone https://github.com/Da-Bois-Tracking/Tracking-App.git`
   `cd tracking-app`
   `cd backend`
2. Make sure to checkout to my feature branch:
   `git checkout TA-2-Create-backend-using-Python`
3. Setup a Virtual Environment
   - For Windows:
      `python -m venv venv`
      `venv/Scripts/activate`
   - For macOS / Linux:
      `python3 -m venv venv`
      `source venv/bin/activate`
4. Install Dependencies:
   `pip install -r requirements.txt`

5. Start the Flask Application:
   - Navigate to the project directory and run:
      `python app.py`
   - The application should be accessible at `http://127.0.0.1:5000/`

6. Verify the Homepage
   - Open a web browser and go to ` http://127.0.0.1:5000/`
   - You should see a "Hello, World!" message. 

### Review Checklist
- [ ] Clone the repository and switch to the feature branch.
- [ ] Follow the installation instructions to set up the project and its dependencies.
- [ ] Verify the Flask application runs successfully and is accessible.
- [ ] Check the homepage or designated route and confirm the expected output.

[Related Jira Ticket](https://bois-tracking-app.atlassian.net/browse/TA-2)
